library/cmake/Modules/FindPHP.cmake

```diff
@@ -87,26 +91,39 @@ MARK_AS_ADVANCED(
 	PHP_VERSION_STRING
 )
 
+if(PHP_FIND_COMPONENTS)
+	foreach(component ${PHP_FIND_COMPONENTS})
+		string(TOUPPER ${component} _COMPONENT)
+		set(PHP_USE_${_COMPONENT} TRUE)
+	endforeach()
+endif()
+
+set(PHP_ZEND_COMPONENT_FOUND FALSE)
+set(PHP_ZEND_COMPONENT_ACCPETED TRUE)
+if(PHP_USE_ZEND_COMPONENT)
+	FIND_PATH(zend_path
```

Comment:
One last thing before merging ;). Could you also rename zend_path to PHP_ZEND_DIR?